### PR TITLE
Point to Zalando organization in a package name

### PR DESCRIPTION
**Pull request by kingasieminiak**
Changed the name of `core-api` package

### CODE REVIEW

**Comment by roflc0pter** [comment]
:+1:

File: packages/core-api/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@backstage/core-api",
+  "name": "@zalando-sunrise/backstage-core-api",
```
Comment (roflc0pter) [inline packages/core-api/package.json:2]: Isn't it called `core` without "api"?
https://www.npmjs.com/package/@zalando-sunrise/backstage-core

Comment (kingasieminiak) [reply packages/core-api/package.json:2]: There are `backstage/core` and  [backstage/core-api](https://www.npmjs.com/package/@backstage/core-api).

**Comment by roflc0pter** [comment]
:+1:

**Review by roflc0pter — APPROVED** [review]

**Comment by kingasieminiak** [comment]
👍

**Pull request merged**